src/core/column/column.factory.spec.js

```diff
@@ -0,0 +1,20 @@
+import {columnFactory} from './column.factory';
```

Comment:
please add test to check body argument

Reply:
let createColumn = columnFactory(model);
let result = createColumn('text', {path: 'foo', 'key': 'bar' etc.});